src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -167,12 +172,36 @@ public String getDisplayName() {
       return "Mesos Cloud";
     }
 
-    // TODO: validate URLs
+    /** Test connection from Mesos Cloud configuration page. */
+    public FormValidation doTestConnection(@QueryParameter("mesosMasterUrl") String mesosMasterUrl) {
+      mesosMasterUrl = mesosMasterUrl.trim();
+
+      if (mesosMasterUrl.startsWith("zk://")) {
+        return FormValidation.warning("Zookeeper paths can be used, but the connection cannot be " +
```

Comment:
This should be done by the actual validation code see https://jira.mesosphere.com/browse/DCOS_OSS-5046.